### PR TITLE
OTA-3308: Add messages for ECU update

### PR DESCRIPTION
**Pull request by mhls**
Signed-off-by: Michal Michalski <ext-michal.michalski@here.com>

### CODE REVIEW

File: ota-plus-web/app/reactapp/src/components/updates/createWizard/Step1.jsx [hunk 1]
```diff
@@ -93,6 +96,7 @@ class Step1 extends Component {
                 })
               )}
             </div>
+            <div className="information-text">{t('updates.creating.wizard.information_step1')}</div>
```
Comment (piotrekc) [inline ota-plus-web/app/reactapp/src/components/updates/createWizard/Step1.jsx]: Where we are checking that user has only one ECU?
Please check the case with many ECUs. This text will be scrolled and placed in the middle of the list.

File: ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailListItem.jsx [hunk 2]
```diff
@@ -185,6 +186,11 @@ class UpdatesWizardDetailListItem extends Component {
             </Col>
           </Form>
         </Row>
+        <Row>
+          <Col span={12}>
+            <div className="information-text">{t('updates.creating.wizard.information_step2')}</div>
```
Comment (piotrekc) [inline ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailListItem.jsx]: We should check how many items we have in the list and show the message only for one item.

File: ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailList.jsx [hunk 3]
```diff
@@ -22,14 +24,23 @@ class UpdatesWizardDetailList extends Component {
...
+          <>
+            {_.map(selectedHardwares, item => (
+              <UpdatesWizardDetailListItem
+                key={item.name}
+                item={item}
+                wizardData={wizardData}
+                onStep2DataSelect={onStep2DataSelect}
+              />
+            ))}
+            {selectedHardwares.length < 2 && (
```
Comment (piotrekc) [inline ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailList.jsx]: Change "selectedHardwares.length < 2" to "selectedHardwares.length === 1"

Comment (mhls) [reply ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailList.jsx]: ok

File: ota-plus-web/app/reactapp/src/components/updates/createWizard/Step1.jsx [hunk 4]
```diff
@@ -92,6 +95,7 @@ class Step1 extends Component {
                   );
                 })
               )}
+              <div className="information-text">{t('updates.creating.wizard.information_step1')}</div>
```
Comment (piotrekc) [inline ota-plus-web/app/reactapp/src/components/updates/createWizard/Step1.jsx]: This should be displayed on when user has one ECU. Additionally it should be not a part of scroll list.

Comment (mhls) [reply ota-plus-web/app/reactapp/src/components/updates/createWizard/Step1.jsx]: done

File: ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailListItem.jsx (hunk 2 above)
Comment (mhls) [reply ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailListItem.jsx]: done

File: ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailList.jsx [hunk 5]
```diff
@@ -22,14 +24,23 @@ class UpdatesWizardDetailList extends Component {
...
+          <>
+            {_.map(selectedHardwares, item => (
+              <UpdatesWizardDetailListItem
+                key={item.name}
+                item={item}
+                wizardData={wizardData}
+                onStep2DataSelect={onStep2DataSelect}
+              />
+            ))}
+            {selectedHardwares.length === 1 && (
```
Comment (piotrekc) [inline ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailList.jsx]: It should be not for selectedHardwares but for SOFTWARES (packages).

Comment (mhls) [reply ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailList.jsx]: in step1 it is checking hardware ids.
Will follow the same pattern here.

Comment (mhls) [reply ota-plus-web/app/reactapp/src/components/updates/createWizard/step2/UpdatesWizardDetailList.jsx]: ah, my mistake, will check with software

File: ota-plus-web/app/reactapp/src/index.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```
Comment (piotrekc) [inline ota-plus-web/app/reactapp/src/index.html]: This file is not needed.

**Pull request merged**